cryoforge/generate.py

```diff
@@ -381,12 +381,29 @@ def create_stac_item(ds, geom, url):
     scene_1_id = ds["img_pair_info"].id_img1
     scene_2_id = ds["img_pair_info"].id_img2
     version = ITS_LIVE_DATA_VERSION
-    path_scene_1 = int(ds["img_pair_info"].attrs.get("path_img1", 0))
-    row_scene_1 = int(ds["img_pair_info"].attrs.get("row_img1", 0))
-    path_scene_2 = int(ds["img_pair_info"].attrs.get("path_img2", 0))
-    row_scene_2 = int(ds["img_pair_info"].attrs.get("row_img2", 0))
-    scene_1_path_row = f"{path_scene_1}/{row_scene_1}" if path_scene_1 != 0 else "N/A"
-    scene_2_path_row = f"{path_scene_2}/{row_scene_2}" if path_scene_2 != 0 else "N/A"
+
+    scene_1_frame = 'N/A'
+    scene_2_frame = 'N/A'
+
+    scene_1_split = scene_1_id.split('_')
+    scene_2_split = scene_1_id.split('_')
+    if mission.startswith('L'):
+        scene_1_frame = scene_1_split[2]
+        scene_2_frame = scene_2_split[2]
+    elif mission.startswith('S1'):
+        scene_1_frame = ds['img_pair_info']['scene_1_frame']
+        scene_2_frame = ds['img_pair_info']['scene_2_frame']
+    elif mission.startswith('S2'):
+        scene_1_frame = scene_1_split[5]
+        scene_2_frame = scene_2_split[5]
+    elif mission.startswith('N'):
+        # CYL_REL_FRM
+        # CYL - Orbit cycle
+        # REL - Relativate orbit track within cycle
+        # FRM - Frame number within orbit track
+        scene_1_frame = f'{scene_1_split[4]}_{scene_1_split[5]}_{scene_1_split[7]}'
+        scene_2_frame = f'{scene_2_split[4]}_{scene_2_split[5]}_{scene_2_split[7]}'
```

Comment:
```suggestion
    elif mission.startswith('N'):
        # REL_FRM
        # REL - Relativate orbit track within cycle
        # FRM - Frame number within orbit track
        scene_1_frame = f'{scene_1_split[5]}_{scene_1_split[7]}'
        scene_2_frame = f'{scene_2_split[5]}_{scene_2_split[7]}'
```
I don't think we want cycle here; it'll increment every 12 days (for up to ~33 years) and would allow us to identify a particular 12-day window but we only care about the relative track+frame so we can identify a whole stack of data.